### PR TITLE
Added filters to summary admin page [ch200]

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/Admin/SummaryAdmin.php [hunk 1]
```diff
@@ -106,6 +118,115 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                     return true;
                 },
             ])
+
```
Comment (illusionOfParadise) [inline src/Admin/SummaryAdmin.php]: extra line

Comment (Remg) [reply src/Admin/SummaryAdmin.php]: good catch

File: src/Entity/MemberSummary/JobExperience.php [hunk 2]
```diff
@@ -128,6 +128,11 @@ class JobExperience implements SummaryItemPositionableInterface
      */
     private $summary;
 
+    public function __toString(): string
+    {
+        return $this->company;
```
Comment (illusionOfParadise) [inline src/Entity/MemberSummary/JobExperience.php]: You use `experiences.company` in the filtre, you don't really need this, do you ?

Comment (Remg) [reply src/Entity/MemberSummary/JobExperience.php]: totally.
it was useful before we decided to split experiences.company and experiences.name

**Pull request merged**